docker-compose/nginx/kuzzle.conf

```diff
@@ -4,8 +4,7 @@ map $http_upgrade $connection_upgrade {
 }
 
 upstream kuzzle {
-{{range $services := service "kuzzle"}}
-  server {{.Address}}:7512;{{else}}  server localhost:80;{{end}}
+  server kuzzle:7512;
```

Comment:
Question: how does it work with multiple kuzzle instances ?

Reply:
Inside the container, `kuzzle` resolves to 3 ips.
nginx is then smart enough to load balance between the fqdn ips .